src/qml/VariableEditor.qml

```diff
@@ -46,7 +46,7 @@ ColumnLayout {
         color: "transparent"
 
         property var itemRow: index
-        property bool canDelete: VariableEditable
+        property bool canDelete: VariableEditable && VariableScope == 0 // application scope
```

Comment:
Ouf, let's do better :) Expose the enum class via Q_ENUM( VariableScope ), then you can do VariableScope == ApplicationScope.

Reply:
Nice :) :+1: